### PR TITLE
darts: add practice exercise

### PR DESCRIPTION
**Pull request by colinleach**
A simple one, to make sure I understand the process.

### CODE REVIEW

**Comment by colinleach** [comment]
One for @ErikSchierboom

File: config.json [hunk 1]
```diff
@@ -485,6 +485,19 @@
...
         ]
+      },
+      {
+        "slug": "darts",
+        "name": "Darts",
+        "uuid": "ddbd717d-a6b5-40fc-9dc8-3526e7409091",
+        "practices": [],
+        "prerequisites": [],
+        "difficulty": 1,
+        "status": "wip",
```
Comment (ErikSchierboom) [inline config.json:496]: I think it's fine to merge this with `active`

```suggestion
        "status": "active",
```

**Review by ErikSchierboom — COMMENTED** [review]
Some minor suggestions. I'll leave the actual R review to @jonmcalder.

File: exercises/practice/darts/.meta/config.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{
+  "authors": [
+    "colinleach"
+  ],
+  "contributors": [
+  ],
```
[the remark this answers is not in the document]
Comment (colinleach) [reply exercises/practice/darts/.meta/config.json]: done

File: config.json [hunk 3]
```diff
@@ -485,6 +485,19 @@
           "loops",
           "math"
         ]
+      },
```
Comment (jonmcalder) [inline config.json:488]: Can we please move this earlier (I'd suggest just after leap) to facilitate recommended order on the website?

**Review by jonmcalder — CHANGES_REQUESTED** [review]
I think this should probably be one of the first few recommended practice exercises, but other than that this PR is great as is - thanks again!

FWIW I bashed out a quick solution in the process of reviewing of this PR and it ended up being the same (literally verbatim) as your examplar solution 😄

File: config.json (hunk 3 above)
Comment (colinleach) [reply config.json]: Changed as requested

**Review by jonmcalder — APPROVED** [review]

**Pull request merged**